src/database/operations/applicationForms/loadApplicationFormBundle.ts

```diff
@@ -2,20 +2,14 @@ import { loadBundle } from '../generic/loadBundle';
 import type { JsonResultSet, Bundle, ApplicationForm } from '../../../types';
 
 export const loadApplicationFormBundle = async (
-	queryParameters: {
-		offset?: number;
-		limit?: number;
-	} = {},
+	offset: number,
+	limit: number,
```

Comment:
This is somewhat nitpicky, but since we're setting the pattern here, I would prefer for the order of the pagination parameters to match [the order they occur in SQL](https://www.postgresql.org/docs/current/queries-limit.html): limit, then offset.

```suggestion
	limit: number,
	offset: number,
```

This is a big change to a small PR, but I think this is when we need to do it!

Reply:
I'm fine making it; though that order always throws me for a loop (since I think in my mind offset is "where to start" and limit is "where to end")

Reply:
If it helps, they're also in alphabetical order! (for these two parameters, at least)